### DISCUSSION TITLE
Suggestion: Making Reset Game Progress into “Achievement Management”

### DISCUSSION THREAD

**Discussion by starblades** [RetroAchievements/RAWeb#1295] [Ideas]
Im not sure if it has been discussed before or not, feel free to close if not relevant. 
BUT, with the talk of resetting games affecting badges in the discord site-feedback. I wanted to draw attention to the section that is the Reset Game Progress in general.
![582B9ADE-243F-48C4-A023-0B23CE7D1EF5](https://user-images.githubusercontent.com/98715022/211096571-f9daa8c7-2f9b-4bcf-974d-2913e6b66deb.jpeg)
I feel like this could be remade into “Achievement Management” or something like that, to make it easier. Adding any buttons you would need. Plus I believe that the Reorder Site Awards should be a part of this as well, because I have seen people confused on where to go to reorder their site awards.
![78C3D28B-8586-4421-B6BF-A8911C3F8EEB](https://user-images.githubusercontent.com/98715022/211096968-e7ddded6-1fa7-46ff-a05b-c0dad6183e52.jpeg)
So I feel like having the achievement management stuff in one section would make it friendlier to newer users on the site and look better in the end.

**Comment by EmoonX** [comment]
Ugh, I missed this discussion before doing #1312.

I like your idea the most, actually. There are too many sections in the Settings page already. Maybe fewer, semantic-grouped sections (subdivided into smaller headers) will achieve a better result than what we have now.

**Reply by GabCandido** [reply]
Maybe leave the left section for user-related stuff:
- Settings
- Avatar
- Notifications
- Keys
- Change Password
- Change Email Address
- Delete Account

While the one on the right for things related to achievements:
- Reset Game Progress
- Request Score Recalculation
- Reorder Site Awards

**Reply by luchaos** [reply]
Heads up: from the V2 prototype at dev.retroachievements.org:

![Screenshot 2023-01-14 at 11 35 30](https://user-images.githubusercontent.com/1280590/212467863-ad5d1bcf-d147-4d74-8629-b729cec2b8ea.png)

Dividing settings into sections, much like here on GitHub, is planned but requires some changes in the templates that I'm currently working on.

"Library" is meant to allow to manage games, achievements, and badges.
